website/docs/addon_applications_admin.md

```diff
@@ -60,4 +70,37 @@ In the image before you can see that we set most of the environment variables in
 In this example MTOA will automatically set the `MAYA_VERSION`(which is set by Maya Application environment) and `MTOA_VERSION` into the `MTOA` variable. We then use the `MTOA` to set all the other variables needed for it to function within Maya.
 ![tools](assets/settings/tools_01.png)
 
-All the tools defined in here can then be assigned to projects. You can also change the tools versions on any project level all the way down to individual asset or shot overrides. So it is possible to upgrade you render plugin for a single shot, while not risking the incompatibilities on the rest of the project.
\ No newline at end of file
+All the tools defined in here can then be assigned to projects. You can also change the tools versions on any project level all the way down to individual asset or shot overrides. So it is possible to upgrade you render plugin for a single shot, while not risking the incompatibilities on the rest of the project.
+
+## Applications filters
+These filters define the applications that will be shown in the launcher tool. It is profile based filtering that allows to filter applications for specific task types.
+
+Using default settings will show all applications for all tasks. That can be easily changed.
+
+Simple example:
+![settings_applications](assets/applications/admin_applications_profiles.png)
+
+In this case all tasks with `Modeling` type will show only `Maya 2025`. Any other task type will show `After Effects 2024`, `Maya 2025`, `Nuke 15.0` and `Photoshop 2024`.
+
+:::note Default studio profile
+It is recommended to have one profile without task types filter, which then acts as the standard set of applications used in the studio.
+:::
+
+## Tools filters
+Tools that will be used on application launch. It is profile based filtering that allows to filter tools for a context. Tools can have more specific filtering than applications.
```

Comment:
It's not a must have - however it **would be nice** if there's also an example of Tool Filters similar to Applications example, including a screenshot.

Anything @MustafaJafar or @LiborBatek could maybe assemble?

Reply:
yeah, I agree. 
We should also mention how the filtering works. 
As far as I know, it uses regex patterns for folder paths. 
anyways, I'll mention it in my review.